src/main/java/org/embulk/output/azure_blob_storage/AzureBlobStorageFileOutputPlugin.java

```diff
@@ -222,10 +222,10 @@ public Void call() throws StorageException, URISyntaxException, IOException, Ret
                                     log.info("Upload start {} to {}", file.getAbsolutePath(), filePath);
                                     blob.upload(new BufferedInputStream(new FileInputStream(file)), file.length());
                                     log.info("Upload completed {} to {}", file.getAbsolutePath(), filePath);
-                                    log.info("Delete completed local file {}", file.getAbsolutePath());
-                                    if (!file.delete()) {
-                                        throw new ConfigException("Couldn't delete local file " + file.getAbsolutePath());
+                                    if (file.exists() && !file.delete()) {
+                                        log.warn("Couldn't delete local file " + file.getAbsolutePath());
                                     }
+                                    log.info("Delete completed local file {}", file.getAbsolutePath());
```

Comment:
This line is logged even when it fails to delete?

Reply:
Thanks. Deleted dde3560